src/scripts.js

```diff
@@ -248,8 +251,10 @@ function addRecipeImage(recipe) {
 }
 
 function generateIngredients(recipe) {
-  return recipe && recipe.ingredients.map(i => {
-    return `${capitalize(i.name)} (${i.quantity.amount} ${i.quantity.unit})`
+  console.log(ingredientsData);
+  return recipe && recipe.ingredients.map(ingredient => {
+    let ingredientInfo = ingredientsData.find(dataIng => dataIng.id === ingredient.id);
+    return `${capitalize(ingredientInfo.name)} (${ingredient.quantity.amount} ${ingredient.quantity.unit})`
```

Comment:
Good fix here! Glad we got the ingredients showing up instead of the recipe name over and over again